tests/baselines/reference/user/TypeScript-React-Starter.log

```diff
@@ -0,0 +1,8 @@
+Exit Code: 2
+Standard output:
+node_modules/@types/enzyme/index.d.ts(368,15): error TS2304: Cannot find name 'Cheerio'.
+node_modules/@types/enzyme/index.d.ts(640,69): error TS2304: Cannot find name 'Cheerio'.
```

Comment:
`@types/enzyme@2` specifies `@types/cheerio@*`, so it gets the newest, incompatible version...

This is a Typescript-react-starter problem, not a user problem. I'll see if we care about upgrading to a newer version of enzyme.